<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/*  ---------------------------- ALL OTHERS WIDGETS ---------------------------- */
*{
selection-background-color: rgb(67, 128, 179);
selection-color: rgb(255, 255, 255);
}

/*  ---------------------------- MAIN WINDOW, WIDGET ---------------------------- */
QMainWindow,QWidget
{
color:rgb(0,0,0);
background:rgb(255,255,255);
}

/*  ---------------------------- MENU BAR ---------------------------- */
QMenuBar
{
color:rgb(120,120,120);
background:rgb(230,230,230);
}
QMenuBar::item:selected
{
color:rgb(100,100,100);
background:rgb(200,200,200);
}
/*  ---------------------------- CONTEXT MENU ---------------------------- */
QMenu
{
color:rgb(90,90,90);
background:rgb(230,230,230);
padding: 3;
}
QMenu::item:selected
{
color:rgb(90,90,90);
background:rgb(200,200,200);
}
QMenu::separator {
background:rgb(200,200,200);
height:1px;
}


/*  ---------------------------- Q TAB BAR ---------------------------- */
QTabBar::tab {
color:rgb(150,150,150);
background:rgb(240,240,240);
height:30;
width:80;
border: rgb(240,240,240);
border-width: 0 0 2px 0;
padding-left:10;
}

QTabBar::tab:selected {
background:white;
color:rgb(100,100,100);
border: solid rgb(0,150,255);
border-width: 0 0 5px 0;
}
QTableWidget QTableCornerButton::section {
background-color: rgb(255, 255, 255);
}
/*-----------------------------------------------------------------
------------------------------ LINE_EDIT --------------------------
------------------------------ TEXT BROWSER -----------------------
------------------------------ TEXT_EDIT---------------------------
------------------------------ PLAIN_TEXT -------------------------
------------------------------------------------------------------- */
QLineEdit,QTextBrowser,QTextEdit,QPlainTextEdit
{
color:rgb(20,20,20);
background-color:white;
border: solid lightgrey;
border-width: 0 0 2px 0;
border-bottom-left-radius: 5;
border-bottom-right-radius: 5;
}

QLineEdit:disabled
{
color:rgb(160, 150, 150);
background-color:rgb(255, 240, 240);
border: solid rgb(253, 14, 14);
border-width: 0 0 2px 0;
border-bottom-left-radius: 5;
border-bottom-right-radius: 5;
}

/*  ---------------------------- COMBO BOX ----------------------------*/
QComboBox
{
color:rgb(0,115,170);
background-color:rgb(255, 255, 255);
min-width: 5px;
padding: 1px 0px 1px 3px;
border: 1px solid rgb(0,115,170);
}

QComboBox:hover
{
color:rgb(0,115,170);
background-color: white;
}

QComboBox:selected
{
color:rgb(0,115,170);
selection-background-color: rgb(255, 255, 255);
}

QComboBox::drop-down
{
width: 30px;
background-color:rgb(0,115,170);
}

QComboBox::down-arrow
{
image: url(assets/UI/Icons/interface_icons/arrow_down.png);
width: 14px;
height: 14px;
}


/* -------------------------------- CHECK BOX ----------------------------------------- */
QCheckBox
{
background: rgb(255, 255, 255);
color:rgb(25, 29, 32);
padding: 6;
}
/* ----------------------------  TOOL BOX  ----------------------------  */
QToolBox::tab
{
color:darkgrey;
background:lightgrey;
}
QToolBox::tab::selected
{
color:grey;
background:rgb(250, 250,250);
}
QToolBox::tab::hover
{
color:white;
background:rgb(0,115,170);
}
/*  ---------------------------- PROGRESS BAR ---------------------------- */
QProgressBar {
color:grey;
text-align: center;
font-size:13px;
}
QProgressBar::chunk {
background:rgb(0, 193, 50);
}
/*  ---------------------------- PUSHBUTTON ---------------------------- */
QPushButton
{
border: 1px solid lightgrey;
color:white;
background:rgb(0,115,170);
min-height:30;
min-width: 50;
}
QPushButton:hover
{
border: 1px solid lightgrey;
color:white;
background:rgb(0, 120, 210);
}

QPushButton:pressed
{
border: 1px solid lightgrey;
color:white;
background:rgb(0, 53, 100);
}
/*  ----------------------------  LCD NUMBER ---------------------------- */
QLCDNumber
{
color:rgb(0,115,170);
border:2 solid rgb(100,100,100);
}
/*  ---------------------------- TABLE_LIST_TABLE ---------------------------- */
QTableView,
QTableWidget
{
alternate-background-color: rgb(240, 250, 255);
}
QTreeView
{
background: rgb(250,250,250);
color: rgb(180,180,180);
}
QTableView::item:selected, 
QListView::item:selected,
QTableView::item:hover, 
QListView::item:hover, 
QTreeView::item:hover
{
background:rgb(0,115,170);
color:rgb(250,250,250);
}
QTableView::item, 
QListView::item, 
QTreeView::item
{
color:rgb(100,100,100);
}
QTreeView::item:selected,QListView::item:selected,QTableView::item:selected
{
color:rgb(37, 62, 71);
background:rgb(209, 241, 252);
}
/* QTreeView::item:has-children
{
background-color: rgb(0, 78, 134);
color: white;
border-bottom: 2px solid qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(255, 255, 255, 0), stop:0.5 rgba(0, 150, 255, 255), stop:1 rgba(255, 255, 255, 0));
}
*/
/*  ---------------------------- HEADER VIEW ---------------------------- */
QHeaderView::section
{
color:rgb(133, 133, 133);
background:white;
border:transparent;
text-align:center;
padding:1;
}
/* ------------------------------- CALENDAR -------------------------------------------------- */
QCalendarView
{
color: rgb(20,20,20);
background-color: rgb(240,240,240);
alternate-background-color: rgb(0,115,170);
selection-background-color: white;
selection-color: black;
}
QAbstractItemView
{
color:rgb(200,200,200);
}


/* ---------------------------------------- SLIDER HORIZONTAL ----------------------------------------------- */


QSlider::groove:horizontal,QSlider::add-page:horizontal
{
background: rgb(255, 255, 255);
height: 27px;
}
QSlider::sub-page:horizontal {
height: 10px;
background: rgb(0,115,170);
}
QSlider::handle:horizontal {
margin-right: -10px;
margin-left: -10px;
background: rgb(0,115,170);
}
QSlider::handle:horizontal:hover {
background:rgb(0,115,170);
}

/* --------------------------------  VERTICAL SLIDER --------------------------------------------------------------  */

QSlider::handle
{
border-radius: 3px;
}

QSlider::groove:vertical,QSlider::add-page:vertical,QSlider::sub-page:vertical
{
width: 20px;
background: rgb(255, 255, 255);
}

QSlider::handle:vertical {
margin-top: -10px;
margin-bottom: -10px;
background: rgb(0,115,170);
}
QSlider::handle:vertical:hover {
background: rgb(0,115,170);
}

/* --------------------------------- SCROLLBAR HORIZONTAL --------------------------------------  */

QScrollBar::groove:horizontal{
background: white;
height: 17px;
}
QScrollBar::sub-page:horizontal,QScrollBar::add-page:horizontal  {
height: 10px;
background: rgb(255, 255, 255);
}
QScrollBar::handle:horizontal {
margin-right: -5px;
background: rgb(0,115,170);
}
QScrollBar::handle:horizontal:hover {
background: rgb(0,115,170);
}

/* --------------------------------------- SCROLLBAR VERTICAL ----------------------------------------------  */

/* SCROLLBAR */
QScrollBar:vertical {
background: white;
width: 15px;
margin: 22px 0 22px 0;
}
/* HANDLE*/
QScrollBar::handle:vertical {
background: rgb(0,115,170);
min-height: 20px;
}

/* UP ARROW */
QScrollBar::up-arrow:vertical {
image: url(assets/UI/Icons/interface_icons/arrow_up.png);
width: 10px;
height: 10px;
}
/* DOWN ARROW */
QScrollBar::down-arrow:vertical {
image: url(assets/UI/Icons/interface_icons/arrow_down.png);
width: 10px;
height: 10px;
}
/* UP BUTTON */
QScrollBar::sub-line:vertical {
background: rgb(0,115,170);
height: 20px;
subcontrol-position: top;
subcontrol-origin: margin;
}
/* DOWN BUTTON */
QScrollBar::add-line:vertical {
background: rgb(0,115,170);
height: 20px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
/* SUBPAGES - ADDPAGE */
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background: none;
}
/* ---------------------------------- Q TOOL BAR -------------------------------------------------------- */
/* TOOLBAR REGION */
QToolBar {
background: rgb(35,40,45);
spacing: 20;

}
/* SEPARATOR */
QToolBar:separator
{
background: rgb(80, 80, 80);
height: 2;
}
/* QToolBar QToolButton { 
    width: 100px;
} */
/* -----------------------------------------------Q TOOL BUTTON------------------------------------------- */
/* BUTTON */
QToolButton
{
color: rgb(255, 255, 255);
background:rgb(35,40,45);
}

QToolButton:hover,QToolButton:pressed
{
background-color: rgb(64, 73, 82);
}

QMessageBox QLabel
{
color: red;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>TEST</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="placeholderText">
          <string>Name </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="placeholderText">
          <string>Code</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>full mark </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>model number</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="placeholderText">
          <string>description file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>....</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="placeholderText">
          <string>path to sheets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Go on</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLineEdit" name="lineEdit_10">
         <property name="placeholderText">
          <string>model answer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>....</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>321</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>5</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Process</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>5</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>username:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>RESULT</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>501</width>
        <height>271</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Analysis</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>256</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>statistic summary</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>256</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>analysis</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionResult">
   <property name="text">
    <string>Result</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
